src/code/macros.asm

```diff
@@ -83,6 +83,26 @@ dialog_pointer: macro
     dw \1
 endm
 
+; Load the low part of a dialog index into the given register.
+; Usage:
+;   ld_dialog_low a, Dialog123 ; ld a, $23
+;
+; NB: if rgbasm ever supports user-defined functions, then we'll replace this by:
+;   ld a, DIALOG_ID_LOW(Dialog123)
+ld_dialog_low: macro
+  ld \1, LOW((\2_IdxOffset - DialogPointerTable) / 2)
+endm
```

Comment:
Great documentation. What does NB stand for?
For the usage doc I would recomend a pure asm to macro comparison.
'ld a, $23' will become 'ld_dialog_low a, Dialog123'

Reply:
NB is [Nota Bene](https://en.wikipedia.org/wiki/Nota_bene) - but that may be a Gallicism :)

I tried to showcase the asm generated code right above ; do you think it should be clearer?

> ; Usage:
> ;   ld_dialog_low a, Dialog123 ; ld a, $23

Reply:
yes. It took me a while. Also the 1 before the 23 could be explained here.

Reply:
I clarified the emitted code.